frontend/src/components/job/info.js

```diff
@@ -23,11 +23,23 @@ class JobInfo extends Component {
                 return null
             })
 
+            let meta = [];
+            let metaTag = '<none>';
+
+            Object.keys(taskGroup.Meta || {}).forEach(function(key) {
+                meta.push(<dt key={key + 'dt'}>{key}</dt>);
+                meta.push(<dd key={key + 'dd'}>{taskGroup.Meta[key]}</dd>);
+            });
+
+            if (meta.length > 0) {
+                metaTag = <dl className="dl-horizontal">{meta}</dl>
+            }
+
             return (
                 <tr key={taskGroup.ID}>
                     <td>{taskGroup.Name}</td>
                     <td>{taskGroup.Count}</td>
-                    <td>{taskGroup.Meta || "<none>" }</td>
+                    <td>{metaTag}</td>
```

Comment:
I've added a more dense version which i really dig :) 

![image](https://cloud.githubusercontent.com/assets/22841/18985985/0d681d1c-86fb-11e6-8eb6-f9695dfdee27.png)

the keys show up as tooltip on hover :)